shared/js/background/https.es6.js

```diff
@@ -178,7 +178,7 @@ class HTTPS {
 
         // Only deal with http calls
         const protocol = utils.getProtocol(reqUrl).toLowerCase()
-        if (!protocol.indexOf('http:') === 0) {
+        if (protocol !== 'http:') {
```

Comment:
The problem here is that `!protocol.indexOf('http:')` evaluated before the equality, so the if statement ended up as either `if (true === 0)` or `if (false === 0)`.

This means we were running the linear search for e.g. HTTPS sites where it wasn't necessary.